### PR TITLE
Update doc to include windows ssh instructions

### PR DESCRIPTION
**Pull request by jbritt1**
Ref: #7044

### CODE REVIEW

**Comment by jbritt1** [comment]
Tagging @department-of-veterans-affairs/vsp-operations

File: platform/engineering/internal-tools.md [hunk 1]
```diff
@@ -90,6 +132,7 @@ here are the steps to create one:
     * You will have a subdirectory in your home directory `~/.ssh`
     * This subdirectory has restrictive permissions `0700`
     * In this subdirectory you will have two files, `id_rsa_vetsgov` and `id_rsa_vetsgov.pub`.
+    * To get the contents of your new public key (which will need to be provided in a future step), you can run `cat ~/.ssh/id_rsa_vetsgov.pub`. This should give a long string of random characters (i.e. ssh-rsa AAAAAjfje983jJL3j2....).
```
Comment (ricetj) [inline platform/engineering/internal-tools.md]: Thank you for adding this.

**Review by ricetj — APPROVED** [review]
Looks good to me and thanks for adding this.

**Comment by jbritt1** [comment]
Good to merge or perhaps should wait on another set of eyes / review you think @ricetj?

**Comment by ricetj** [comment]
> Good to merge or perhaps should wait on another set of eyes / review you think @ricetj?

Yeah, it would be good to have another set of eyes.

**Review by nathanhruby — APPROVED** [review]

**Pull request merged**